protocol/src/main/java/org/opensearch/sql/protocol/response/QueryResult.java

```diff
@@ -32,6 +32,9 @@ public class QueryResult implements Iterable<Object[]> {
    */
   private final Collection<ExprValue> exprValues;
 
+  @Getter
+  private final String cursor;
```

Comment:
we should make sure backwards compatible with legacy cursor implementation.

Reply:
Noted